modules/http/server/src/test/scala/GreeterRestTests.scala

```diff
@@ -114,83 +117,135 @@ class GreeterRestTests extends RpcBaseTestSuite with BeforeAndAfter {
   "REST Service" should {
 
     "serve a GET request" in {
-      val response = (for {
+      val response = for {
         client   <- Http1Client[IO]()
         response <- unaryServiceClient.getHello()(client)
-      } yield response).unsafeRunSync()
-      response shouldBe HelloResponse("hey")
+      } yield response
+      response.unsafeRunSync() shouldBe HelloResponse("hey")
     }
 
     "serve a unary POST request" in {
       val request = HelloRequest("hey")
-      val response = (for {
+      val response = for {
+        client   <- Http1Client[IO]()
+        response <- unaryServiceClient.sayHello(request)(client)
+      } yield response
+      response.unsafeRunSync() shouldBe HelloResponse("hey")
+    }
+
+    "handle a raised gRPC exception in a unary POST request" in {
```

Comment:
Why would we have a gRPC exception in this case?

Reply:
Great question! I should have addressed this in the patch notes. Basically if we want the service handlers to be able to server both RPC and REST requests more or less transparently we'll want consistency in the error reporting. I re-used the GRPC exceptions for backwards compatibility since we already have support for them in RPC handlers; but there's also a good case to be made in providing exceptions of our own that would be interpreted to grpc exceptions or REST statuses depending on the context.